starport/cmd/network_chain_publish.go

```diff
@@ -47,16 +53,28 @@ func NewNetworkChainPublish() *cobra.Command {
 
 func networkChainPublishHandler(cmd *cobra.Command, args []string) error {
 	var (
-		source        = args[0]
-		tag, _        = cmd.Flags().GetString(flagTag)
-		branch, _     = cmd.Flags().GetString(flagBranch)
-		hash, _       = cmd.Flags().GetString(flagHash)
-		genesisURL, _ = cmd.Flags().GetString(flagGenesis)
-		chainID, _    = cmd.Flags().GetString(flagChainID)
-		campaign, _   = cmd.Flags().GetUint64(flagCampaign)
-		noCheck, _    = cmd.Flags().GetBool(flagNoCheck)
+		source            = args[0]
+		tag, _            = cmd.Flags().GetString(flagTag)
+		branch, _         = cmd.Flags().GetString(flagBranch)
+		hash, _           = cmd.Flags().GetString(flagHash)
+		genesisURL, _     = cmd.Flags().GetString(flagGenesis)
+		chainID, _        = cmd.Flags().GetString(flagChainID)
+		campaign, _       = cmd.Flags().GetUint64(flagCampaign)
+		noCheck, _        = cmd.Flags().GetBool(flagNoCheck)
+		rewardCoinsStr, _ = cmd.Flags().GetString(flagRewardCoins)
+		rewardDuration, _ = cmd.Flags().GetUint64(flagRewardHeight)
 	)
 
+	rewardCoins, err := sdk.ParseCoinsNormalized(rewardCoinsStr)
+	if err != nil {
+		return err
+	}
+
+	if (!rewardCoins.Empty() && rewardDuration == 0) ||
+		(rewardCoins.Empty() && rewardDuration > 0) {
+		return errors.New("--reward.coins and --reward.duration flags must be provided together")
```

Comment:
Can we use var names instead e.g. flagRewardCoins?

Reply:
Yes, but I think this is way more user-friendly. The developer may not know what `flagRewardCoins` is

Reply:
What I meant is something like this:

```go
fmt.Errorf("%s and %s flags must be provided together", flagRewardCoins, flagRewardDuration)
```